very easy/29.js

```diff
@@ -1,8 +1,8 @@
 function calculateExponent(num, exp) {
-	
+  return Math.pow(num, exp);

```

Comment:
In newer version of ES6, we can use `**` operator to exponent numbers

Reply:
ok, i changed -> num ** exp